.env

```diff
@@ -0,0 +1,8 @@
+DG_API_KEY="ecec76b6cd2c0b75d45500734c4fb37e2be6f09e"
```

Comment:
Hmmm... why did this get pushed up? I see .env in the .gitignore file. 

I'll kill all these api keys. We should delete this

Reply:
I noticed it later on in one of the other PRs and removed it there. This should get taken care of down the line.